node/src/Transport.ts

```diff
@@ -1246,14 +1246,26 @@ export class Transport
 			throw new TypeError('every type must be a string');
 		}
 
+		// Convert event types.
+		const fbsEventTypes: FbsTransport.TraceEventType[] = [];
+
+		for (const eventType of types)
+		{
+			try
+			{
+				fbsEventTypes.push(transportTraceEventTypeToFbs(eventType));
+			}
+			catch (error) { /* Ignore invalid event types */ }
```

Comment:
can we please?

can we please?

```suggestion
			try
			{
				fbsEventTypes.push(transportTraceEventTypeToFbs(eventType));
			}
			catch (error)
			{
				// Ignore invalid event types.
			}
```